package/MDAnalysis/core/Selection.py

```diff
@@ -910,7 +850,6 @@ class SelectionParser:
     CYZONE = 'cyzone'
     POINT = 'point'
     BYRES = 'byres'
-    BONDED = 'bonded'
```

Comment:
See comment above: just comment out this line and reference the issue for re-implementation in the comment.